schema.graphql

```diff
@@ -58,3 +58,39 @@ type NFT @entity {
   "Absorbed NFTs"
   absorbedNFTs: [NFT!]!
 }
+
+"""
+The NFT collection, merge., and inherent global details on it.
+"""
+type Collection @entity {
```

Comment:
You can make all `Int`s on this type `Int!`

Reply:
Agreed. One of those things I was planning to clean up after getting it all to work! Will make the change.